flax/nn/base.py

```diff
@@ -398,6 +398,10 @@ def create(cls, _rng, *args, name=None, **kwargs):
       **kwargs: keyword arguments passed to the module's apply function
     Returns:
       A pair consisting of the model output and an instance of Model
+    Example:
+      input_shape = (batch_size, image_size, image_size, 3)
+      model_output, initial_params = model.init_by_shape(jax.random.PRNGKey(0),
```

Comment:
`init_by_shape` is the counterpart of `create_by_shape`, and using `init` is the alternative of using `create`. 
But regardless of that,  both `create` and `create_by_shape` are going to be removed soon and since they are deprecated, we left no pointer to these two in the examples or documentation to fade them out.

Reply:
Thank you for review! 
but sorry, I write the doc to wrong place :/ I corrected now. Do you have any templates for writing usage examples, etc..